src/sage/doctest/external.py

```diff
@@ -495,5 +503,28 @@ def seen(self):
                 for feature, seen in zip(self._features, self._seen)
                 if seen > 0]
 
+    def hidden(self):
+        """
+        Return the list of detected hidden external software.
+
+        EXAMPLES::
+
+            sage: from sage.doctest.external import available_software
```

Comment:
```suggestion
            sage: # needs conway_polynomials database_cremona_mini_ellcurve database_ellcurves database_graphs
            sage: from sage.doctest.external import available_software
```

Reply:
Sorry, I did it by myself, because I missed that I just need to commit your suggestion! Thanks!